src/pages/Home.js

```diff
@@ -1,9 +1,16 @@
 import React, { Component } from 'react'
+import cuteasscat from '../assets/cuteasscat.png'
 
 class Home extends Component{
   render(){
     return(
-      <h1>Welcome to cat tinder!</h1>
+      <>
+      <h1><strong>Welcome to Cat Tinder! 
```

Comment:
Please clean up the indentation on this page.

Reply:
Done